Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -3769,3 +3769,28 @@ def test_warnings_handler(mocker):
     msg = demisto.info.call_args[0][0]
     assert 'This is a test' in msg
     assert 'python warning' in msg
+
+
+class TestCommonTypes:
```

Comment:
it seems as though these classes. are being tested in separate UT, e.g: `test_create_domain`, `test_create_dbot_score_with_invalid_reliability`, `test_create_certificate`, `test_multiple_indicators`.
it is a good idea to create a class for them IMO, but please move all of them to be in the same logical place, as we want this _test file to be easy to navigate through.